ctapipe/image/cleaning.py

```diff
@@ -93,3 +93,39 @@ def dilate(geom, mask):
         input mask (array of booleans) to be dilated
     """
     return mask | geom.neighbor_matrix_sparse.dot(mask)
+
+
+def number_of_islands(geom, mask):
+    """
+    Search a given pixel mask for connected clusters.
+    This can be used to seperate between gamma and hadronic showers.
+    Returns total number of clusters and an integer numpy array.
+    Array entries correspond to cluster assignment, bool of array 
+    corresponds to input pixel mask.
+
+    Parameters
+    ----------
+    geom: `~ctapipe.instrument.CameraGeometry`
+        Camera geometry information
+    mask: ndarray
+        input mask (array of booleans)
+    """
```

Comment:
Documentation of output Parameters is missing

Reply:
True
Will add that asap